google/cloud/bigquery/version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__version__ = "2.1.0"
```

Comment:
@crwilcox Does it make sense to set this to the next version now? Or will the release bots find this and update the version number in that PR?

Reply:
Now that I release 2.1.0, maybe this is correct?

Reply:
I wouldn't bump this in advance. @crwilcox submitted a PR for release-please to find and bump this I believe.

Reply:
Yep. Release please will figure it out. Though you may have to close the first release it opens and force trigger. The way we find files is the index, and version.py won't exist at first 👍

Reply:
@crwilcox ` version.py won't exist at first` means version is black or empty string in very first time? or should i erase it from file?

Reply:
@HemangChothani version.py won't exist in the index of repository files. I was commenting on release infrastructure, not the source. The file should exist and the version should be the same as it was in setup.py.